inference-engine/src/transformations/src/transformations/op_conversions/convert_gather_downgrade.cpp

```diff
@@ -39,3 +41,29 @@ ngraph::pass::ConvertGather7ToGather1::ConvertGather7ToGather1() {
     auto m = std::make_shared<pattern::Matcher>(gather_v7, matcher_name);
     register_matcher(m, callback);
 }
+
+ngraph::pass::ConvertGather8ToGather7::ConvertGather8ToGather7() {
+    MATCHER_SCOPE(ConvertGather8ToGather7);
+
+    auto gather_v8 = pattern::wrap_type<ngraph::opset8::Gather>();
+
+    ngraph::matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto gather_v8_node = std::dynamic_pointer_cast<ngraph::opset8::Gather>(m.get_match_root());
+        if (!gather_v8_node)
+            return false;
+
+        auto data_input = gather_v8_node->input_value(0);
+        auto indices_input = gather_v8_node->input_value(1);
+        auto axis_input = gather_v8_node->input_value(2);
+        auto batch_dims = gather_v8_node->get_batch_dims();
+
+        auto gather_v7 = std::make_shared<ngraph::opset7::Gather>(data_input, indices_input, axis_input, batch_dims);
+        gather_v7->set_friendly_name(gather_v8_node->get_friendly_name());
+        ngraph::copy_runtime_info(gather_v8_node, gather_v7);
+        ngraph::replace_node(gather_v8_node, gather_v7);
+        return true;
```

Comment:
there is some duplicate with `ConvertGather7ToGather1`? I think we can combine these transformations

Reply:
Generalized these transformations, this reduces duplication. Please review